homeassistant/components/opnsense/const.py

```diff
@@ -0,0 +1,9 @@
+"""Constants for OPNsense component."""
+
+DATA_HASS_CONFIG = "opnsense_hass_config"
+DOMAIN = "opnsense"
+OPNSENSE_DATA = DOMAIN
+
+CONF_API_SECRET = "api_secret"
+CONF_INTERFACE_CLIENT = "interface_client"
+CONF_TRACKER_INTERFACE = "tracker_interfaces"
```

Comment:
Consider renaming:

```suggestion
CONF_TRACKER_INTERFACES = "tracker_interfaces"
```

(will require refs to be updated)